server/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "seed": "ts-node ./seed",
+    "server": "nodemon server"
+  },
+  "keywords": [],
+  "author": "",
+  "dependencies": {
+    "apollo-datasource": "^0.7.3",
+    "apollo-server": "^2.19.2",
+    "dotenv": "^8.2.0",
+    "eslint": "^7.19.0",
```

Comment:
Noticed that there's no `.eslintrc.json`. Are we planning on doing the eslint set up in a different PR?

Reply:
Did the setup for it - I'm not familiar with what options are there. I think it'd be best if someone else finishes configuring it

Reply:
Totes, thanks for setting it up! I'll take care of the rest of the config.

Not an eslint expert, but I'm going to try to match the client eslint config as closely as possible.